token_embedding.py

```diff
@@ -0,0 +1,75 @@
+# this is a word embedding module using bert
+
+from bert_embedding import BertEmbedding
+from spacy.pipeline import SentenceSegmenter
+from spacy.lang.en import English
+import numpy
+import scipy
+
+class BertEmbedder:

```

Comment:
Are you planning on using this class instead of the default word2vec implementation in find_similar.py?